modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -227,53 +226,55 @@ rpk topic alter-config <topic> --set segment.ms=<segment_ms_duration>
-* Log storage (`retention_local_target_capacity_percent` or `retention_local_target_capacity_bytes`): This log data reservation is the disk space used as the target maximum size for user data, as well as Redpanda internal topics, like the control log. It's generally about 70-80% of total disk space.
+Space management divides the disk space into different areas that can be managed separately:
+
+* Reserved disk space (`disk_reservation_percent`) is the area of disk space that Redpanda does not use.
+** As the disk space used by the other areas grows to their target sizes, this provides a buffer space to avoid free disk space alerts.
+** SSDs that run near capacity can experience performance degradation, so this buffer space prevents the disks from running at capacity.
+* Cache storage (`cloud_storage_cache_size_percent` or `cloud_storage_cache_size`) is the maximum size of the xref:../tiered-storage.adoc#caching[disk cache] used by Tiered Storage. As the cache reaches its limit, new data added to the cache removes old data from the cache.
+* Log storage (`retention_local_target_capacity_percent` or `retention_local_target_capacity_bytes`) is the area of disk space used for topic data. This is typically 70-80% of total disk space.
 
 image::shared:disk_storage.png[Redpanda disk storage categories]
 
-When log data usage begins to approach the target size of log storage, data is removed from local disk according to an eviction policy that follows cluster-level and topic-level retention settings. When log data usage exceeds its configured target size, Redpanda selects data to remove to bring usage back under the target size. Redpanda attempts to be fair with one round-robin removal at a time of a segment across partitions that are eligible to have segments removed. Data removal occurs in each phase. As soon as storage usage falls below the target, the data removal process ends.
+Log segment eviction occurs in each of the following phases. As soon as log storage usage falls below the target, the eviction process ends.
 
 See also: xref:manage:tiered-storage.adoc#object-storage-housekeeping[Object storage housekeeping]
 
 === Phases of data removal
 
-==== 1: Follow retention policy
+==== Phase 1: Follow retention policy
 
-A periodic housekeeping task in Redpanda performs compaction and removes data that has expired according to your retention policy. This process applies to both Tiered Storage and non-Tiered Storage topics. When the target size is reached, Redpanda prefers removal of expired data over compaction, and attempts to apply retention to partitions in the order that removes the largest amount of data.
+A housekeeping process in Redpanda periodically performs compaction and removes partition data that has expired according to your retention policy. This applies to both Tiered Storage and non-Tiered Storage topics. Space management attempts to apply retention to partitions in the order that removes the largest amount of data.
```

Comment:
@dotnwat @ztlpn - I'm not 100% settled on the descriptions for phase 1 and 2. 

Do you think it that it makes sense to say that the log storage reservation is treated as a cache when `retention_local_strict` is false, and when true it reverts back to the topic-level retention settings to remove expired data?